packages/components/src/box-control/test/index.js

```diff
@@ -124,6 +124,24 @@ describe( 'BoxControl', () => {
 			expect( input.value ).toBe( '' );
 			expect( unitSelect.value ).toBe( 'px' );
 		} );
+
+		it( 'should persist cleared value when focus changes', () => {
+			const { container } = render( <BoxControl /> );
+			const input = container.querySelector( 'input' );
+			const unitSelect = container.querySelector( 'select' );
+
```

Comment:
Would the following also work?

```js
const input = screen.getByLabelText( 'Box Control', {
				selector: 'input',
			} );
const unitSelect = screen.getByLabelText( 'Select unit' );
```

Noting after a dogmatic reading of [React Testing Library's docs](https://testing-library.com/docs/queries/about#manual-queries).

😄

Reply:
Oh, yes — good to be in keeping with the suggested way of doing it (I was just copying and pasting from other tests 😁). I've updated this one in https://github.com/WordPress/gutenberg/pull/39109/commits/106dfa816dc6ad1755cf26c438ac83086ec84e96, but left the others as-is to keep the size of this PR contained (happy to follow-up on the others separately, though, if we think it's worth it!)